projects/data_generation/synthetic-multi-view-facial-image-generation/README.md

```diff
@@ -0,0 +1,51 @@
+# Synthentic Multi-view Facial Image Generation based on Rotate-and-Render: Unsupervised Photorealistic Face Rotation from Single-View Images (CVPR 2020)
+
+[[Paper]](https://arxiv.org/abs/2003.08124)
+
+We utilize with small modifications  (in order to be easily executed) publicly available code, namely an  un-supervised framework that can synthesize 
+photorealistic rotated facial images using as input  a single facial image, or multiple such images (one per person). The key insight of the utilized method is that rotating faces in the 3D space back and forth, 
+and re-rendering them to the 2D plane can serve as a strong self-supervision.
+
+#Sources:
+* Face Alignment in Full Pose Range: A 3D Total Solution (IEEE TPAMI 2017)
+* Neural 3D Mesh Renderer (CVPR 2018)
+* Rotate-and-Render: Unsupervised Photorealistic Face Rotation from Single-View Images (CVPR 2020)
+## Requirements
+* Python 3.6 is used. Basic requirements are listed in the 'requirements.txt'.
+
+```
+pip3 install -r requirements.txt
+```
+* Install the [Neural_Renderer](https://github.com/daniilidis-group/neural_renderer) following the instructions.
+```
+pip3 install neural_renderer_pytorch
+```
+
+* Download checkpoint and BFM model from [ckpt_and_bfm.zip](https://cicloud.csd.auth.gr/owncloud/remote.php/webdav/OpenDR/FTP%20Server%20Material/simulation/ckpt_and_bfm.zip) put it in ```3ddfa``` and unzip it. The 3D models are borrowed from [3DDFA](https://github.com/cleardusk/3DDFA). 
+
+
+## DEMO
+
+#1. Download the [checkpoint](https://cicloud.csd.auth.gr/owncloud/remote.php/webdav/OpenDR/FTP%20Server%20Material/simulation/latest_net_G.zip)
```

Comment:
Just a suggestion, if you remove the "#" this will automatically generate an enumeration.